doc/Install.md

```diff
@@ -145,6 +145,16 @@ Example:
 
 *Note*: Do **not** rename the .htaccess-dist file as it is tracked by GIT and renaming will cause a dirty working directory.
 
+### Verify the "host-meta" page is working
+
+Friendica should respond automatically to an important address named /.well-known/host-meta  This URL would look like, for example, https://example.com/.well-known/host-meta  It must be visible to the public and must respond with an XML file that is automatically customized to your site.
+
+If that URL is not working, it is possible that some other software is using the /.well-known/ path and is not configured correctly to work with Friendica.  Other symptoms may include an error message in the Admin settings that says "host-meta is not reachable on your system. This is a severe configuration issue that prevents communication."  Another common error related to host-meta is the, "Invalid profile URL."
```

Comment:
Can you please add line breaks after every sentence you've added to the docs. It is the one rule to ease the process of maintaining the Markdown files (e.g. to spot far too long sentences) and it makes the diffs more easy to read. As Markdown is interpreted, those newlines won't be seen in the browser.